src/routes/_components/LengthIndicator.html

```diff
@@ -42,11 +44,12 @@
     store: () => store,
     computed: {
       lengthToDisplay: ({ length, max }) => max - length,
-      lengthLabel: ({ overLimit, lengthToDisplayDeferred }) => {
-        if (overLimit) {
-          return formatIntl('intl.overLimit', { count: -lengthToDisplayDeferred })
+      lengthVerbosity: ({ lengthToDisplayDeferred }) => {
+        // When approaching the limit, notify screen reader users
+        if (lengthToDisplayDeferred > WARN_THRESHOLD) {
```

Comment:
```suggestion
        if (lengthToDisplayDeferred > WARN_THRESHOLD || lengthToDisplayDeferred <= 0) {
```

@nvdaes It seems to me that we should keep warning if the user reaches 0 or goes beyond the limit (-1, -2, etc.). Let me know what you think of this change. :arrow_up: